src/call/mod.rs

```diff
@@ -45,31 +45,66 @@ pub struct GrpcByteBuffer {
     pub raw: *mut grpc_sys::GrpcByteBuffer,
 }
 
-impl Default for GrpcByteBuffer {
-    fn default() -> Self {
+impl GrpcByteBuffer {
+    /// Both parameters marked as mutable since the ref count increases.
+    pub fn push(&mut self, slice: &mut GrpcSlice) {
+        unsafe { grpc_sys::grpcwrap_byte_buffer_add(self.raw as _, slice) }
+    }
+
+    pub fn pop(&mut self) {
```

Comment:
Deadcode?

Reply:
I think this should be useful in the future. It's not much work (it's an API already provided by gRPC) to add it, I just kept it.